src/components/AudioInfo.vue

```diff
@@ -1177,13 +977,28 @@ const adjustSliderValue = (
   justify-content: flex-start;
   gap: 0px 0;
   overflow-y: scroll;
+
+  &.is-in-audio-detail {
+    width: 100%;
+
+    .parameters {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      padding: 0 16px;
+      gap: 16px;
+    }
+  }
+}
+
+.parameters {
+  flex-grow: 1;
+  padding: 0 16px;
 }
```

Comment:
cssよく知らないんですが、たぶん上書きする要素は近くに書いたほうがメンテナンス性が高い気がします！
```suggestion
  .parameters {
    flex-grow: 1;
    padding: 0 16px;
  }

  &.is-in-audio-detail {
    width: 100%;

    .parameters {
      display: grid;
      grid-template-columns: 1fr 1fr;
      padding: 0 16px;
      gap: 16px;
    }
  }
}
```
（めっちゃ細かいですが）